Restore always_expand support [CUSTOM]

nicoe opened the pull request:
The call to expand_row must occur in redraw_async in order to happen on every row even the one that are never shown.

The call must happen before the redraw call as this call might itself make async calls.

PCLAS-581

jmousset commented:
from:

![cluttered](https://user-images.githubusercontent.com/16104512/98102031-58933400-1e93-11eb-86d9-c3bff84285a3.png)

to:

![Uploading sous_menu.png…]()

PierreCookie reviewed (approved):
Yep c'est pe mieux :D à bp dans tryton ?

nicoe committed:
Restore always_expand support [CUSTOM]

The call to expand_row must occur in redraw_async in order to happen on
every row even the one that are never shown.

The call must happen before the redraw call as this call might itself
make async calls.

JCavallo commented:
Ko sur nos cas à nous

nicoe committed:
Expand all rows after a delay

Change the API of expand_row to return a promise to ensure all the
expansions are done before recomputing if there are other expansions to
do.

nicoe committed:
The delay does not need to be this big.

JCavallo commented:
Solution non satisfaisante en l'état, à remettre à plat à l'occasion
